fenix/app/src/main/java/org/mozilla/fenix/compose/ListItemTabLargePlaceholder.kt

```diff
@@ -37,19 +38,21 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * ```
  *
  * @param text The only [String] that this will display.
+ * @param backgroundColor The background [Color] of the story.
```

Comment:
```suggestion
 * @param backgroundColor The background [Color] of the list item placeholder.
```
nit: This is suppose to be a generic component.